src/settings-ui/Settings.UI/SettingsXAML/Views/PowerAccentPage.xaml

```diff
@@ -63,6 +63,7 @@
                             <ComboBoxItem x:Uid="QuickAccent_SelectedLanguage_Dutch" />
                             <ComboBoxItem x:Uid="QuickAccent_SelectedLanguage_Greek" />
                             <ComboBoxItem x:Uid="QuickAccent_SelectedLanguage_Estonian" />
+                            <ComboBoxItem x:Uid="QuickAccent_SelectedLanguage_Esperanto" />
```

Comment:
The resource text string in https://github.com/microsoft/PowerToys/blob/main/src%2Fsettings-ui%2FSettings.UI%2FStrings%2Fen-us%2FResources.resw is missing. Please add it.